symfit/core/fit_results.py

```diff
@@ -10,36 +15,52 @@ class FitResults(object):
     - R squared (Regression coefficient.) or other fit quality qualifiers.
     - fitting status message
     - covariance matrix
+    - objective and minimizer used.
 
     Contains the attribute `params`, which is an
     :class:`~collections.OrderedDict` containing all the parameter names and
     their optimized values. Can be `**` unpacked when evaluating
     :class:`~symfit.core.fit.Model`'s.
     """
-    def __init__(self, model, popt, covariance_matrix, infodic, mesg, ier, **gof_qualifiers):
+    @keywordonly(constraints=None)
+    def __init__(self, model, popt, covariance_matrix, minimizer, objective, **minimizer_output):
```

Comment:
My current problem with this signature is that `minimizer_ouput` *must* contain a `'message'` and `'infodic'`, which is rather obscure to figure out. I suggest to explicitly add those to to the signature.

Reply:
Good point, I've added `message` to the signature. However, `infodic` is something SciPy specific, so I have actually removed that as a `property`. It is still accessible through `minimizer_output` but I think that is good enough.